ads/google/a4a/performance.js

```diff
@@ -270,4 +281,56 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
   getInitTime() {
     return this.initTime_;
   }
+
+  /**
+   * Adds CSI pings for various visibility measurements on element.
+   *
+   * @param {!Element} element Amp ad element we are measuring.
+   * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+   * @param {!Object} signals Signals object for element.
+   * @override
+   */
+  addPingsForVisibility(element, ampDoc, signals) {
```

Comment:
Few words about arguments:

1. `ampDoc` is not needed - you can simply pass the `element` to `analyticsForDoc(element, true)`.
2. Why pass `signals`. Isn't this always `element.signals()`?